Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:stretch
```

Comment:
Not really, just picked it because that's mostly what we settled on for most of the docker library images